src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line
+import React, { Component } from 'react';
+// eslint-disable-next-line
+export default class Calculator extends React.Component {
+// eslint-disable-next-line
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks, please fix this to ensure you follow best practices 👍🏿

Reply:
Hello @prolajumokeoni I made this so that it can pass react linters , It is asking me to use `this` key word on render